rbac/management/permission/view.py

```diff
@@ -93,3 +98,41 @@ def list(self, request, *args, **kwargs):
         }
         """
         return super().list(request=request, args=args, kwargs=kwargs)
+
+    @action(detail=False)
+    def options(self, request):
+        """Get options of applications."""
+        """
+        @api {get} /api/v1/permissions/options/   Get option of permission
+        @apiName getPermissionOption
+        @apiGroup Permission
+        @apiVersion 1.0.0
+        @apiDescription Get option of permission
+
+        @apiHeader {String} token User authorization token
+
+        @apiParam (Query) {String} field The field to return.
+
+        @apiParam (Query) {String} application Filter by permission name.
+        @apiParam (Query) {String} resource_type Filter by resource_type.
+        @apiParam (Query) {String} verb Filter by verb.
+
+        @apiSuccess {Object[]} data  The array of results.
+
+            HTTP/1.1 200 OK
+        {
+          "data": [
+            catalog, approval
+          ]
+        }
+        """
+        filters = {}
+        query_field = request.query_params.get(QUERY_FIELD, "application")
```

Comment:
This works, but can only handle returning a single field at a time. Is there any possibility that we'd need to return multiple fields with a request like /permissions/?field=application,verb&verb=read or something? Not sure if this is a viable use case or not.

Reply:
Yeah, I was wondering the same with regard to having multiple values like `?field=application,verb`.

I also feel like we should have some validation on this to prevent 500s when you supply an unsupported `field` value?

Reply:
Yea, I would add a validation to the field. Multiple fields would be nice to have but temporary I can't think of a use case for that. Maybe we could leave that as a future enhancement?  

At first what they want is returning options for all the fields (application, resource_type, verb). Then I looked at their use case, they just want to that when customer click e.g. verb, get all the verbs and pop the list (this PR will make that possible), then customer could select one verb and then return all the permissions that have the verb (currently supported).
Then I discussed with Robin, maybe it is better to support when customer click e.g. application, get all application, then, customer select one of them, e.g., 'catalog', then when they click the resource_type, then they would get all the resource_type that belong to 'catalog' pop up.
This way, 
1. UI is satisfied for getting one field options;
2. we don't have to return all the fields, especially, when the available options become large, we could reduce the payload.